README.md

```diff
@@ -53,11 +53,10 @@ brew info autoenv
 You can then copy the `.env_SAMPLE` file to `.env`, then update any environment variables accordingly.
 
 ### Dependencies
-This project uses `requirements.txt` files for defining dependencies, so you
-can get up and running with `pip`:
+This project uses `setup.py` for defining dependencies. You can get up and running with `pip`:
 
 ```shell
-$ pip install -r requirements.txt       # modules required for execution
+$ pip install .                         # modules required for execution
```

Comment:
We should recommend using `pip install -e` as that will install the local modules as ["editable"](https://pip.pypa.io/en/stable/reference/pip_install/#editable-installs), which provides for a better local development experience. Perhaps also say something about using a virtualenv to avoid installing the packages globally?

Additionally, a good practice is to add the testing dependencies into an "extra" section in the setup.py (see below). Then the recommended command would be what you have in the tox.ini: `pip install -e '.[testing]'`.